@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1648,7 +1685,7 @@ export class TextElementsRenderer {
 
         // Render the label's text...
         // textRenderState is always defined at this point.
-        if (renderText) {
+        if (renderText && textCanvas) {
```

Comment:
Better shortcircuit the call to shouldRenderPointText:

const renderText = textCanvas && shouldRenderPointText(labelState, this.m_viewState, this.m_options);

Reply:
done